lib/exp.js

```diff
@@ -762,10 +762,6 @@ exports.var = (varName) => [
  * LIST READ EXPRESSIONS
  *********************************************************************************/
 
-function getContextType (ctx, type) {
```

Comment:
I think this function is required, and needs to be implemented to match the logic in the corresponding C function here: https://github.com/aerospike/aerospike-client-c/blob/master/src/main/aerospike/as_exp.c#L425-L435

Reply:
Also means you need unit tests to cover the different context types used in list expressions.